.env.example

```diff
@@ -1,6 +1,9 @@
 MOUNT_FOLDER=/tmp/pdf_dir
 MAX_FILE_SIZE_MB=123
 ENVIRONMENT=local
+
+# Variable not present in src/settings.py as it is only used in diff scripts
+# It will still be read by the settings.py file
```

Comment:
The code looks good now, but I'm unsure about this comment. It will be ignored, not read, thanks to the `extra='allow'` parameter, as I understand it. Maybe just delete this line?

Reply:
Correct, its not read by the settings class at all. This environment variable is read with config from decouple in the diff scripts, so I've removed the comment.